extras/speaker-recognition/charts/templates/secret.yaml

```
@@ -0,0 +1,18 @@
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-secrets
+  labels:
+    app: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+type: Opaque
+data:
+  {{- if .Values.secrets.hfToken }}
+  HF_TOKEN: {{ .Values.secrets.hfToken | b64enc | quote }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)